target/linux/tcall.c

```diff
@@ -196,6 +198,54 @@ static long _tcall_get_tsd(uint64_t* value)
     return 0;
 }
 
+long myst_tcall_impersonate(long n, long params[6], uid_t uid, gid_t gid)
```

Comment:
Suggest renaming to `myst_tcall_with_identity`.

Reply:
renamed